news/2024/2024-11-25-the-followpoint-hivie-the-man-of-anything-and-everything.md

```diff
@@ -0,0 +1,354 @@
+
+---
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/avatars/MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+Oki doki, here comes my favorite part, medals :3
+What are your thoughts on the current state of medals?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/14102976" style="background-image: url('/wiki/shared/avatars/Hivie.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-nat" href="https://osu.ppy.sh/users/14102976">Hivie</a></p>
+
+I think we've come a long way with medals, which is kind of surprising considering how limited the scope of medals is with osu!(stable). However I feel like we've started to plateau in terms of creativity and variety, that's why I'm really looking forward when we start releasing osu!(lazer)-exclusive medals. The possibilities there are much, *much* more vast with the gigantic library of mods, and how feasible is it to integrate medals in other areas technically (i.e. multiplayer).
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/avatars/MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+What do you like about developing them the most?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/14102976" style="background-image: url('/wiki/shared/avatars/Hivie.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-nat" href="https://osu.ppy.sh/users/14102976">Hivie</a></p>
+
+osu! medal hunters are insane schizos (in a good way), and trying to match that energy when brainstorming medals is always fun.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/avatars/MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+Speaking of development, do you have any ETA on when medals will display which map you received them on?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/14102976" style="background-image: url('/wiki/shared/avatars/Hivie.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-nat" href="https://osu.ppy.sh/users/14102976">Hivie</a></p>
+
+Honestly I'm not sure, It might go against the spirit of secrecy with Hush-Hush medals, so it will need further considerations by the osu! team.
```

Comment:
```suggestion
Honestly I'm not sure, It might go against the spirit of secrecy with Hush-Hush medals, so it will need further consideration by the osu! team.
```